frappe/contacts/doctype/address/address.py

```diff
@@ -81,7 +77,7 @@ def has_common_link(self, doc):
 		return False
 
 @frappe.whitelist()
-def get_default_address(doctype, name, sort_key='is_primary_address'):
+def get_default_address(doctype, name, sort_key='is_primary_address', existing_address=None):
```

Comment:
Why are you passing an argument just to get the same value back? Why not patch at the client side to not call this or verify the address directly there.